src/lib/containers/table/EntityIDColumnCopyIcon.tsx

```diff
@@ -32,13 +45,17 @@ const EntityIDColumnCopyIcon = () => {
     queryRequestClone.query = {
       sql: `select id from ${entityId}${versionNumberString}`,
     }
-    getFullQueryTableResults(queryRequestClone, synapseContext.accessToken)
+    getFullQueryTableResults(
+      queryRequestClone,
+      synapseContext.accessToken,
+      abortController?.signal,
+    )
       .then((data: QueryResultBundle) => {
         setIdData(data)
       })
       .catch((err: any) => {
         console.error(err)
-        displayToast('Unable to query for all IDs', 'danger')
+        displayToast('Unable to query for all Synapse IDs to copy', 'danger')
       })
```

Comment:
If the query is cancelled, the `AbortError` would be caught here, and the toast would be shown. Seems fine to me.